tests/Fixtures/themes/LegacyTestTheme/TestBundle/translations/messages.en.yml

```diff
@@ -0,0 +1,3 @@
+test:
+#    legacy_theme: 'THEME/TestBundle/translations'
```

Comment:
`tests/Fixtures/themes/LegacyTestTheme/translations/messages.en.yml` should not be overridden by this one. It works correctly locally on macOS, however it doesn't work on Travis. I've got no idea why it is the case, as the order in which they're loaded stays the same on both environments.

I'd just document it in known issues and don't care too much about this edge case (which can be easily solved, as both directories are parts of the same theme).